code/modules/cargo/bounties/silly.dm

```diff
@@ -0,0 +1,43 @@
+/datum/bounty/item/silly/paper_plane
+	name = "Paper Planes"
+	description = "Centcom is running an experiment on which type of paper plane is the most aerodynamically stable. Send us some samples to further our study."
+	reward = 900
+	required_count = 15
+	wanted_types = list(/obj/item/paperplane)
+
+/datum/bounty/item/silly/mousetraps
+	name = "Mouse Traps"
+	description = "Some shoeless hoodlum is intentionally tracking paint all over Centcom. Send us some mouse traps so we can capture the culprit."
+	reward = 1200
+	required_count = 12
+	wanted_types = list(/obj/item/assembly/mousetrap)
+
+/datum/bounty/item/silly/action_figures
+	name = "Action Figures"
+	description = "The vice president's son saw an ad for action figures on the telescreen and now he won't shut up about them. Ship some to ease his complaints."
+	reward = 4000
```

Comment:
Way to high for 300 plastic 2500 would be better

Reply:
The reason it's so high is because you need to have they toy lathe researched (except on Delta) in order to complete it. This takes a significant amount of time and resources. This was the default number when I moved it from Assistant Bounties to Silly Bounties.